src/shadowbox/docker/build.action.sh

```diff
@@ -21,7 +21,16 @@ export DOCKER_BUILDKIT=1
 # Newer node images have no valid content trust data.
 # Pin the image node:16.18.0-alpine3.16 by hash.
 # See image at https://hub.docker.com/_/node/tags?page=1&name=16.18.0-alpine3.16
-readonly NODE_IMAGE="node@sha256:264861cd2f785a2b727e9f908065e8d9e9358fcc1308da3cb207d9cba69afee2"
+readonly NODE_IMAGE=$(
+    if [[ "$(uname -m)" == "x86_64" ]]; then
```

Comment:
Instead of automatically detecting, can you make the architecture be an input instead? I'd like to be able to cross-compile, like I do in https://github.com/Jigsaw-Code/outline-sdk/blob/main/CONTRIBUTING.md#building

I guess we can use an env var, which the default being `x86_64`

Reply:
Thanks for your time and sensible comments.
I added ARCH environment variable.